crates/starknet-devnet-types/src/rpc/contract_class/deprecated/rpc_contract_class.rs

```diff
@@ -77,10 +77,16 @@ impl HashProducer for DeprecatedContractClass {
 
 impl TryInto<CompressedLegacyContractClass> for DeprecatedContractClass {
     type Error = Error;
+
     fn try_into(self) -> Result<CompressedLegacyContractClass, Self::Error> {
-        // TODO: improve
```

Comment:
I don't think the `TODO: improve` here referred to improving the clarity of the code, but rather to improve this conversion chain. But I'm not the author of this `TODO`.